tensorflow/python/debug/wrappers/hooks.py

```diff
@@ -31,15 +31,17 @@
 class LocalCLIDebugHook(session_run_hook.SessionRunHook):
   """Command-line-interface debugger hook.
 
-  Can be used as a monitor/hook for `tf.train.MonitoredSession`s and
-  `tf.contrib.learn`'s `Estimator`s and `Experiment`s.
+  Can be used as a monitor/hook for `tf.train.MonitoredSession`s,
+  `tf.estimator.Estimator`s, and `tf.contrib.learn`'s `Estimator`s and
+  `Experiment`s.
```

Comment:
I believe `Experiment` is deprecated. Please remove this reference. cc @ispirmustafa

Reply:
Upon checking I discovered contrib.learn.Estimator is also deprecated. Will remove both.